tokio/src/runtime/builder.rs

```diff
@@ -294,6 +300,13 @@ impl Builder {
         self
     }
 
+    /// Disable tokio's automated cooperative scheduling mechanism See this link for details:
+    /// <https://tokio.rs/blog/2020-04-preemption>
+    pub fn no_coop(&mut self) -> &mut Self {
+        self.coop_budget = Budget::unconstrained();
+        self
+    }
```

Comment:
I only had a quick look. Is this the only change in the public API?

In any case, it needs better docs.

Reply:
There is this and a similar change in LocalSet. I agree this needs more docs — I'm happy to add the polish here as soon as we agree on the general direction :) 

Any guidance on the kind of documentation this warrants? The underlying feature didn't have that much I could link to, but let me know what you think?

Reply:
Yeah, making sure we agree on the general direction first is probably good.

Regarding the kind of doc, it could look something like this:

> Disable the automated cooperative scheduling feature.
>
> The coop feature limits the number of operations that can be performed before a task yields to the executor. You can read more about the coop feature in the blog post [Reducing tail latencies with automatic cooperative task yielding][1].
>
> ### Example
>
> ```Rust
> use tokio::runtime;
>
> let rt = runtime::Builder::new_multi_thread()
>     .enable_all()
>     .disable_coop()
>     .build()
>     .unwrap();
> ```

[1]:  https://tokio.rs/blog/2020-04-preemption